data/migrations/versions/34c8ef052ec9_repo_mirror_columns.py

```diff
@@ -89,6 +89,8 @@ def upgrade(op, tables, tester):
 
     logger.info("Migrating to external_reference from existing columns")
     op.add_column("repomirrorconfig", sa.Column("external_reference", sa.Text(), nullable=True))
+    if op.get_context().dialect.name != "sqlite":
+        op.get_bind().execute("COMMIT")
```

Comment:
It is worth adding a comment on why COMMIT is required for all DBs except SQLite